CHANGELOG.md

```diff
@@ -1,5 +1,98 @@
+  
+  Previously using standard GitHub packaged zips as download source would not work as, even though the plugin would be installed, it would not be recognized as such by TGMPA because of the non-standard directory name which would be created for the plugin, i.e. `my-plugin-master` instead of `my-plugin`. A work-around for this has been implemented and you can now use GitHub-packaged `master` branch or release zips to install plugins. Have a look at the `example.php` file for a working example.
+
+  One caveat: this presumes that the plugin is based in the root of the GitHub repo and not in a `/src` subdirectory.
+
+  [#327], [#280], [#283] Thanks [Dan Fisher] and [Luis Martins] for reporting/requesting this enhancement.
+
+* Enhancement: **New optional plugin parameter `is_callable`**.
+
+  Some plugins may have a free and a Premium version using different slugs. Using the `is_callable` plugin parameter allows for the Premium version to be recognized, even though the slug is set to the free version slug. Have a look at the `example.php` file for a working example.
+
+  For more information on what is considered a `callable`, please refer to the [Codex on callbacks].
+  
+  [#205] Props [Zack Katz].
+
+* **Admin Page improvements**:
+  - Plugins downloaded from an arbitrary external source are now labelled _"External Source"_. Previously they were labelled _"Private Repository"_ which could be confusing as the download URL did not have to point to a repository, let alone be private. [#372]
+  - Leverage the css styling of the Core standard `WP_List_Table` [#227]. Props [Shiva Poudel].
+  - Allow for moving the Admin Page to a different place in the menu. This is mainly to accommodate plugins using TGMPA as it is terribly illogical for the TGMPA page to be under the _"Appearance"_ menu in that case. This has been now been implemented in a way that Theme Check will not choke on it. [#310]
+
+* **Admin notices improvements**:
+  - For installs with both plugin(s) as well as theme(s) using TGMPA, notices will now be dismissable for each separately. This prevents a situation where a theme would have TGMPA included, the user has dismissed the notice about it, a plugin with TGMPA gets installed and the notice about it requiring certain other plugins is never shown. [#174] Thanks [Chris Howard] for reporting.
+  - Fixed: The reset of dismissed notices on `switch_theme` was only applied for the current user, not for all users. [#246]
+  - Fixed: Admin notices would show twice under certain circumstances. [#249], [#237] Thanks [manake] for reporting.
+
+* **Bulk Installer**:
+  - Fixed: If a bulk install was initiated using the bottom _"Bulk Actions"_ dropdown, the install page would display an outdated TGMPA plugin table at the bottom of the page after the bulk installation was finished. [#319]
+  - Fixed: The _"Show Details"_ links no longer worked. This was a regression briefly introduced in the `develop` branch. [#326]
```

Comment:
Agreed, I was in dubio about that one and erred on cautious side. This was just round the release of 2.4.1, so if people mistakenly had downloaded `develop` instead of 2.4.1, this one could be in the wild.
Now I'm writing this I'm actually in doubt that it wasn't included in 2.4.1 - let me check...